### PR TITLE
operators universal-crossplane (1.6.1-up.1)

### PR DESCRIPTION
**Pull request by turkenh**
Thanks submitting your Operator. Please check below list before you create your Pull Request.

### New Submissions

* [ ] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-via-pr.md)?
* [ ] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [ ] Have you tested your Operator with all Custom Resource Definitions?
* [ ] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [ ] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
* [ ] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?

### Updates to existing Operators

* [x] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [x] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [x] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [ ] Have you tested an update to your Operator when deployed via OLM?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [x] Modify more than one operator
* [x] Modify an Operator you don't own
* [x] Rename an operator - please remove and add with a different name instead
* [x] Modify any files outside the above mentioned folders
* [x] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [x] Description about the managed Application and where to find more information
2. [x] Features and capabilities of your Operator and how to use it
3. [x] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [x] Human readable name and 1-liner description about your Operator
* [x] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [x] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [x] Links to the maintainer, source code and documentation
* [x] Example templates for all Custom Resource Definitions intended to be used
* [x] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

**Review by muvaf — APPROVED** [review]
LGTM!

**Comment by turkenh** [comment]
TL;DR:
`Operator test / kiwi / Full operator test` failing,  do we have to remove `ServiceAccount` manifests from the bundle to have it passed?

--- 
Check for operator validation fails with the following: 

```
    time="2022-01-20T14:27:20Z" level=error msg="Error: Value crossplane: invalid service account found in bundle. sa name cannot match service account defined for deployment spec in CSV"
    time="2022-01-20T14:27:20Z" level=error msg="Error: Value rbac-manager: invalid service account found in bundle. sa name cannot match service account defined for deployment spec in CSV"
    time="2022-01-20T14:27:20Z" level=error msg="Error: Value upbound-bootstrapper: invalid service account found in bundle. sa name cannot match service account defined for deployment spec in CSV"
    time="2022-01-20T14:27:20Z" level=error msg="Error: Value xgql: invalid service account found in bundle. sa name cannot match service account defined for deployment spec in CSV"
```

On my local, with operator-sdk `v1.16.0`, I am getting similar but not exactly same errors:

```
ERRO[0000] Error: Value crossplane: invalid service account found in bundle. This service account crossplane in your bundle is not valid, because a service account with the same name was already specified in your CSV. If this was unintentional, please remove the service account manifest from your bundle. If it was intentional to specify a separate service account, please rename the SA in either the bundle manifest or the CSV.
ERRO[0000] Error: Value rbac-manager: invalid service account found in bundle. This service account rbac-manager in your bundle is not valid, because a service account with the same name was already specified in your CSV. If this was unintentional, please remove the service account manifest from your bundle. If it was intentional to specify a separate service account, please rename the SA in either the bundle manifest or the CSV.
ERRO[0000] Error: Value upbound-bootstrapper: invalid service account found in bundle. This service account upbound-bootstrapper in your bundle is not valid, because a service account with the same name was already specified in your CSV. If this was unintentional, please remove the service account manifest from your bundle. If it was intentional to specify a separate service account, please rename the SA in either the bundle manifest or the CSV.
ERRO[0000] Error: Value xgql: invalid service account found in bundle. This service account xgql in your bundle is not valid, because a service account with the same name was already specified in your CSV. If this was unintentional, please remove the service account manifest from your bundle. If it was intentional to specify a separate service account, please rename the SA in either the bundle manifest or the CSV.
```

I've found this issue: https://github.com/operator-framework/operator-sdk/issues/5244

There are some back and forth discussions, and I noticed someone removed SA manifests from bundle but later added back etc.

Should we remove as well? Is this just a workaround or should we always drop them while generating the bundle?

**Comment by mvalarh** [comment]
In pipeline we are using now `operator-sdk 1.15.0` (https://github.com/k8s-operatorhub/community-operators/runs/4883428385?check_suite_focus=true#step:6:155). And i think workaround would help. We are going to upgrade to latest `operator-sdk` soon and i hope it will help

**Review by negz — APPROVED** [review]

**Comment by J0zi** [comment]
@turkenh are you able to install the operator locally? You can debug using https://github.com/redhat-openshift-ecosystem/community-operators-pipeline/blob/documentation/docs/operator-test-suite.md

**Comment by J0zi** [comment]
@turkenh  closing due to no activity, feel free to reopen any time.
